### DISCUSSION TITLE
Additional Data on Flows

### DISCUSSION THREAD

**Discussion by Tschulius94** [FlowmapBlue/FlowmapBlue#56] [Ideas]
Hi,
I am trying to create a map of Coffee flowing from all over the world to the roaster in Germany. Right now I am using the count value as the number of coffeebags(70kg) that are being shipped over the world. It would be cool to add addintional values like additional coffee details and the price to the flows beeing displayed at the tooltip.
Still this far.
Super cool tool.

**Comment by ilyabo** [comment]
Hi Tschulius94, 
Sorry, missed to answer your message earlier. This is a good suggestion. However, it might be difficult to make it work with the dynamic flow aggregation due to clustering and time range selection. I will think about it.
